test/data_frame_test.exs

```diff
@@ -2,7 +2,76 @@ defmodule DataFrameTest do
   use ExUnit.Case
   doctest DataFrame
 
-  test "the truth" do
-    assert 1 + 1 == 2
+  alias DataFrame.Frame
+
+  def empty_frame do
+    %Frame{values: [[]], index: [], columns: []}
+  end
+
+  def single_entry_frame do
+    DataFrame.new([[0, 0]], [:A])
+  end
+
+  describe "new/2" do
+    test "empty input is an empty Frame" do
+      # should the index be [[]]? Shouldn't it be []?
+      assert DataFrame.new([[]], []) == 
+        %Frame{values: [[]], index: [[]], columns: []}
```

Comment:
recent change in master now it match with `%Frame{values: [[]], index: [], columns: []}` sorry for the bug.